menu/src/components/misc/HelpTooltip.tsx

```diff
@@ -1,11 +1,17 @@
 import React, { ReactNode } from "react";
-import { Fade, Theme, Tooltip, Typography } from "@mui/material";
-import makeStyles from "@mui/styles/makeStyles";
+import {Fade, styled, Theme, Tooltip, Typography} from "@mui/material";
```

Comment:
color is bad, also not bouncing but this is not super important.  
Feel free to remove bouncyness if you wish.